### PR TITLE
format all files

### PR DESCRIPTION
**Pull request by juniorbutyeah**

### CODE REVIEW

**Comment by xbubbo** [comment]
Update your PR to include my recent changes.

**Comment by xbubbo** [comment]
@juniorbutyeah

File: index.js [hunk 1]
```diff
@@ -11,24 +13,45 @@ const bareServer = createBareServer('/outerspace/');
...
 app.use(express.static(path.join(__dirname, 'static')));
 
-const routes = [
-  { path: '/', file: 'index.html' },
-  { path: '/news', file: 'apps.html' },
-  { path: '/events', file: 'games.html' },
-  { path: '/diagnostic', file: 'settings.html' },
-  { path: '/local-news', file: 'tabs.html' },
-  { path: '/image-galleries', file: 'go.html' },
+const routes = [{
```
[the remark this answers is not in the document]
Comment (juniorbutyeah) [reply index.js:22]: womp womp my way is better

File: index.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import express from 'express';
 import http from 'node:http';
-import { createBareServer } from "@tomphttp/bare-server-node";
+import {
+  createBareServer
```
[the remark this answers is not in the document]
Comment (juniorbutyeah) [reply index.js:4]: denesting imports is my favorite thing to do, mald

File: index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ypxa) [reply index.js:22]: ew thats nasty

**Review by ypxa — COMMENTED** [review]
ew thats nasty

**Comment by xbubbo** [comment]
@juniorbutyeah @VillainsRule @ypxa 

I just made a commit that formatted all of the code using prettier https://github.com/InterstellarNetwork/Interstellar/commit/1a192aa2f3b2c819f85cbaaa14b29e774fbf02fd, if you guys would like to make any changes go for it.

**Pull request closed without merging**